source/server/config/network/http_connection_manager.cc

```diff
@@ -85,6 +85,36 @@ HttpConnectionManagerConfig::HttpConnectionManagerConfig(const Json::Object& con
     use_remote_address_ = config.getBoolean("use_remote_address");
   }
 
+  if (config.hasObject("forward_client_cert")) {
+    // TODO: Check mTLS.
+    const std::string forward_client_cert = config.getString("forward_client_cert");
```

Comment:
Can you write this as `const std::string forward_client_cert = config.getString("forward_client_cert", Http::ForwardClientCertType::Sanitize);` to simplify default case handling?